main.js

```diff
@@ -18,4 +19,285 @@ function close() {
+                    <li class="background1"><a class="textdecoration font4" href="#">${cardsData[i].technologiesDesk[2]}</a></li>
+                    <li class="background1"><a class="textdecoration font4" href="#">${cardsData[i].technologiesDesk[3]}</a></li>
+
+                </ul>
+
+                <button class="font5 textdecoration seepopup" type="button" index=${cardsData[i].id}>See Project</button>
+
+            </div>
+
+        </article>
+
+  `;
+}
+
+worksSection.innerHTML = '';
+worksSection.innerHTML = displayCard;
+
+const showPopupBtn = document.querySelectorAll('.seepopup');
+const popUpContainer = document.createElement('section');
+popUpContainer.className = 'popup_section';
+popUpContainer.classList.add('bg_color6');
+showPopupBtn.forEach((e) => {
+  e.addEventListener('click', () => {
+    const btnIndex = e.getAttribute('index');
+
+    const popupEle = `
+  <article class="popup_article bg_color7">
+
+<div class="popup_articlediv show1">
+
+    <header class="popup_flex1">
+
+        <h3><a class="textdecoration font1" href="#">${cardsData[btnIndex].name[0]}</a></h3>
+
+        <button type="button" class="close_popup "><img src="images/close.png" alt=""></button>
+
+    </header>
+
+    <ul class="list1">
+
+        <li><a class="textdecoration font2" href="#">CANOPY</a></li>
+        <li><a class="textdecoration" href="#"><img src="images/Counter.png" alt="mit_dot"></a></li>
+        <li><a class="textdecoration font3" href="#">Back End Dev</a></li>
+        <li class="textdecoration"><a href="#"><img src="images/Counter.png" alt="mit_dot"></a></li>
+        <li><a class="textdecoration font3" href="#">2015</a></li>
+
+
+    </ul>
+
+    <figure class="figure1">
+
+        <a href="#"><img src="${cardsData[btnIndex].image[0]}" alt="Tonic_img"></a>
+
+    </figure>
+
+    <article class="desc1">
+        <p>${cardsData[btnIndex].description[0]}</p>
+
+        <ul class="gridbox1 gridbox1b">
+
+            <li class="background1"><a class="textdecoration font4" href="#">${cardsData[btnIndex].technologies[0]}</a></li>
+            <li class="background1"><a class="textdecoration font4" href="#">${cardsData[btnIndex].technologies[1]}</a></li>
+            <li class="background1"><a class="textdecoration font4" href="#">${cardsData[btnIndex].technologies[2]}</a></li>
+
+        </ul>
+    </article>
+
+    <footer>
+        <button class="footer_btn textdecoration" id="see-projects-btn" type="submit">
+            <a href="${cardsData[btnIndex].liveLink}"><span>See Live</span></a>
+            <img src="images/live.png" alt="">
+        </button>
+
+        <button class="footer_btn textdecoration" id="see-projects-btn" type="submit">
+        <a href="${cardsData[btnIndex].Source}"><span>See Source</span></a>
+            <img src="images/github.jpg" alt="">
+        </button>
+        
+    </footer>
+
+</div>
```

Comment:
- [ ] Kindly make sure your popup design sticks as much as possible to the one from the Figma template such as **background color**, **Blur filter effect** **image height**,... Try to check all information from **[Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=oH5WI0fQMPSvteiz-0)** and improve your design
![Screenshot from 2023-02-10 14-39-35](https://user-images.githubusercontent.com/83042742/218095507-91865ab4-2bf5-49e2-84c0-a490c978c168.png)